dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -708,4 +774,70 @@ private Integer uninstall() {
         }
         return OK;
     }
-}
+
+    private static void getShortNameFromManifest(File esa, Map<String, String> shortNameMap) throws IOException {
+        String esaLocation = esa.getAbsolutePath();
+        ZipFile zip = null;
+        try {
+            zip = new ZipFile(esaLocation);
+            Enumeration<? extends ZipEntry> zipEntries = zip.entries();
+            ZipEntry subsystemEntry = null;
+            while (zipEntries.hasMoreElements()) {
+                ZipEntry nextEntry = zipEntries.nextElement();
+                if ("OSGI-INF/SUBSYSTEM.MF".equalsIgnoreCase(nextEntry.getName())) {
+                    subsystemEntry = nextEntry;
+                    break;
+                }
+            }
+            if (subsystemEntry == null) {
+                ;
+            } else {
```

Comment:
Just use `if (subsystemEntry != null) { Manifest m ...... }`

Reply:
This change results in error: Only one "/" symbol is allowed in the resourceString, do you have alternate suggestions? Individual esa install works when this change is reverted.